DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/Acquirente/Carrello.tex

```diff
@@ -56,7 +56,7 @@
     \begin{enumerate}[label=\lett]
         \item L'attore modifica la quantità con una minore o uguale a zero. In questo caso:
         \begin{itemize}
-            \item (UC\ref{estensione:quantita-minore-a-zero}) - Viene visualizzato il messaggio d'errore quantità del prodotto non valida;
+            \item (UC\ref{estensione:quantita-non-valida}) - Viene visualizzato il messaggio d'errore quantità del prodotto non valida;
```

Comment:
Però questa è l'estensione "Visualizzazione messaggio d'errore quantità da aggiungere al carrello non valida", mentre il prodotto qua è già dentro il carrello

Reply:
ero in dubbio anch'io all'inizio ma Ho pensato che andasse bene lo stesso perché ho inteso che anche se modifichi è comunque un'aggiunta al carrello di prodotti... se è troppo in dubbio modifichiamo il testo/nome se no bisogna c'è tutta la numerazione degli uc da cambiare se ne aggiungiamo un altro